configs/config.yaml

```diff
@@ -3,11 +3,17 @@ tag: default
 debug: False
 load_data_from_pkl: 0
 load_model: 0
+save_model: 0
 input_model: 0
 #output path is set in code to hydra's current working directory
 #variable is then used downstream to set paths.
 output_path: ???
 create_plots: 0
+train: 1
+test: 0
+device: gpu
+gpu_list:
```

Comment:
Hey @abhishekabhishek , gonna start a quick review. It would be great if you could fill me in on these points.

The `gpu_list` here is a numbered list of gpu ids to use? Like `0,1,3,5` if there are 6 gpus available on the machine? Do you know whether these `ids` are always number from 0 to X?

Reply:
Yes. gpu_ids are always from 0 to X. You can use nvidia-smi to check the memory usage on the GPUs before you start training to avoid memory overflow situations.

Reply:
Currently, only single GPU training is supported but can be extended to multiple GPUs if needed.